CONTRIBUTING.md

```diff
@@ -1,95 +1,37 @@
-# How to Contribute
+# Contributing to the InnerSource Patterns
 
-First, checkout our complete list of patterns: [public GitHub](https://github.com/InnerSourceCommons/InnerSourcePatterns#reviewed-patterns-proven-and-reviewed) OR [Google Doc](https://docs.google.com/spreadsheets/d/17KPZdCoquTnYSj03pX4v2vn8lrSYO_6HK20u1cwaLPg/edit#gid=0)
+The goal of the InnerSource patterns working group is to collect, document, and publish InnerSource best practices. To make the best practices easy to comprehend, evaluate, and apply, we codify them in a specific structure - [the patterns format](meta/pattern-template.md). More infos on the working group can be found in our [README.md](README.md) file.
 
-We encourage beginners seeking answers to jump in by creating `donut` patterns (filling in the problem, context, forces and resulting context fields but leaving the solution blank) as a way of asking the InnerSource Commons community for help (to find a proven solution or to brainstorm things to try). 
+For this working group to strive, we welcome your contribution - be it small or huge. 
 
-Anyone can offer reviews and comments for [in-progress patterns](https://github.com/InnerSourceCommons/InnerSourcePatterns/pulls). We encourage experts to pad their experience - these are hoped to become part of an Inner Source handbook one day.
 
-We work together via GitHub, WebEx, Slack, etc. Do not hesitate to join the [#innersourcecommons](https://isc-inviter.herokuapp.com/) or #innersource-patterns Slack channels and ask to be included in the [patterns meetings](/meta/meetings.md) (there is an email list).
+## How to Contribute?
 
-Select one of the following ways to contribute, to learn our workflow:
+Please consider contributing in the following ways:
 
-* A. [Write a new pattern](#a-writing-a-new-pattern)
-* B. [Discuss/Record early ideas](#b-discussing-early-ideas-in-issues)
-* C. [Review existing patterns](#c-interacting-with-patterns-reviews)
-* D. [Take part in our Meetings and Roles](/meta/meetings.md) <-- link to separate doc
+1. **Read & improve existing patterns.** If you want to get started, we encourage you to read through the existing patterns and materials in this repository. Found a misleading phrase or phony grammar? A mistake in the content? Let us know by [creating an issue](https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/new) or directly propose your fix as a pull request.
```

Comment:
Not sure I can make commit suggestions. It seems requiring write access to the upstream repo. 
> If the pull request was created from a fork and the author allowed edits from maintainers, you can also apply suggested changes if you have write access to the upstream repository.

Reply:
> Not sure I can make commit suggestions. It seems requiring write access to the upstream repo.
> 
> > If the pull request was created from a fork and the author allowed edits from maintainers, you can also apply suggested changes if you have write access to the upstream repository.

Thanks for finding that out @fwan2000. Maybe @maxcapraro can help here as he is the original author of the pull request?

Reply:
My goal with this list was to focus on `what` (as in "What can I contribute?") and then let the `how` (as in "How can I achieve that?") follow as part of the list items. My hope is that this speaks to potential contributors who have a goal in mind already (`what`) and want to hear the specific way to do it ( `how)`.

I agree though, with @fwan2000 that the importance of creating issues for discussing things is not prominent enough in the contributing.md. So, I added it also to point 4.

@fwan2000: If that is okay for you, please mark this comment as resolved.

Reply:
@maxcapraro Thanks for clarifying your thoughts on the separation of `what` and `how`. I meant to include a type of contribution as `sharing a thought or asking a question to spark a dialog in the community`. I feel it's more free form contribution that helps to attract new ideas. Since it's a free form, I was suggesting doing that as an issue instead of a PR. Does that make sense to you?

Reply:
Allright! Now I understand you better. This is missing from the current version, indeed :) I'll add it.

Reply:
I added a text sinppet inviting individuals to `Share your thoughts & spark discussion`

Reply:
@maxcapraro The change looks great! Thanks for incorporating that. I don't seem able to mark the comment resolved. Please go ahead to resolve this. Thanks!